new-lamassu-admin/src/pages/LoyaltyPanel/IndividualDiscounts.js

```diff
@@ -96,44 +95,38 @@ const IndividualDiscounts = () => {
       width: 312,
       textAlign: 'left',
       size: 'sm',
-      view: t => {
-        const customer = getCustomer(t.customerId)
-        return (
-          <div className={classes.identification}>
-            <PhoneIdIcon />
-            <span>{customer.phone}</span>
-          </div>
-        )
-      }
+      view: discount => (
+        <div className={classes.identification}>
+          <PhoneIdIcon />
+          <span>{discount.phone}</span>
+        </div>
+      )
     },
     {
       header: 'Name',
       width: 300,
       textAlign: 'left',
       size: 'sm',
-      view: t => {
-        const customer = getCustomer(t.customerId)
-        if (R.isNil(customer.idCardData)) {
-          return <>{'-'}</>
-        }
-
-        return (
-          <>{`${customer.idCardData.firstName ?? ``}${
-            customer.idCardData.firstName && customer.idCardData.lastName
-              ? ` `
-              : ``
-          }${customer.idCardData.lastName ?? ``}`}</>
+      view: discount =>
+        R.isNil(discount.idCardData) ? (
+          <>{'-'}</>
+        ) : (
+          <>
+            {R.join(' ', [
+              discount.idCardData.firstName ?? '',
+              discount.idCardData.lastName ?? ''
+            ])}
```

Comment:
I know this was just a formatting change, but there's `formatFullName()` in `new-lamassu-admin/src/utils/customer.js` to do this, that takes care of a few edge cases too. Can you change it pls?

Reply:
Yeah, ofc. I didn't know we had that.